app/src/main/java/com/projectfkklp/saristorepos/managers/UserStoreRelationManager.java

```diff
@@ -45,6 +46,31 @@ public static void request(Context context, Store store, UserRole userRole, OnSe
         });
     }
 
+    public static void invite(Context context, User user, UserRole userRole, OnSetFirebaseDocument onSetFirebaseDocument){
+        String id = ModelUtils.createUUID();
+        UserStoreRelation userStoreRelation = new UserStoreRelation(
+                id,
+                SessionRepository.getCurrentUser(context).getId(),
+                user.getId(),
+                userRole,
+                UserStatus.REQUESTED
+        );
```

Comment:
Check the detail from store recruitment perspective, change the code

```java
        UserStoreRelation userStoreRelation = new UserStoreRelation(
                id,
                user.getid(),
                SessionRepository.getCurrentStore(context).getId(),
                userRole,
                UserStatus.INVITED
        );
```

Reply:
![image](https://github.com/Fr4nzy/SariStorePOS/assets/132080920/8d6b3cf2-0565-4ba0-b240-b6c5c4076de0)